setup.cfg

```diff
@@ -35,8 +35,8 @@ setup_requires = setuptools>=62.4
 python_requires = >=3.7
 
 [options.entry_points]
-distutils.commands =
+setuptools.commands = 
     clean_rust=setuptools_rust:clean_rust
     build_rust=setuptools_rust:build_rust
-distutils.setup_keywords =
+setuptools.setup_keywords =
```

Comment:
It looks to me like these have not changed; I suggest reverting these changes (or alternatively raising PRs to setuptools to rename these).

Reply:
Good catch :+1: